grpc/examples/src/main/scala/sttp/tapir/grpc/examples/GrpcSimpleBooksExample.scala

```diff
@@ -0,0 +1,151 @@
+package sttp.tapir.grpc.examples
+
+import akka.actor.ActorSystem
+import akka.grpc.ProtobufSerializer
+import akka.http.scaladsl.Http
+import akka.util.ByteString
+import cats.implicits._
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.StrictLogging
+import sttp.tapir.DecodeResult.Value
+import sttp.tapir._
+import sttp.tapir.generic.auto._
+import sttp.tapir.grpc.protobuf._
+import sttp.tapir.grpc.protobuf.model._
+import sttp.tapir.grpc.protobuf.pbdirect._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.akkahttp.AkkaHttpServerInterpreter
+
+import scala.concurrent.Future
+
+case class SimpleBook(title: String)
+
+/** Descriptions of endpoints used in the example.
+  */
+object Endpoints {
+
+  private val baseEndpoint = endpoint.errorOut(stringBody)
+
+  val addBook = endpoint
+    .in("Library" / "AddBook")
+    .in(grpcBody[SimpleBook])
```

Comment:
for the future: it would be great to be able to define an endpoint which works both with json and grpc. Sth like: `oneOfBody(jsonBody[Book], grpcBody[Book])`. The grpc body probably defines a unique content-type, so this shouldn't get in the way of  normal http endpoints (due to content negotiation).